### ISSUE TITLE
Move all tests into one describe.

### ISSUE THREAD

**Issue by raq929** [ga-wdi-boston/node-api-diagnostic#127]
Instead of four describes with the same text.

**Comment by gaand** [comment]
I agree the test scheme could be improved, but I'm not sure this is the right answer. 

The error test at least seems like it belongs in a separate describe.
